src/chainer.test.js

```diff
@@ -1 +1,56 @@
+'use strict';
+
 const { chainer } = require('./chainer');
+
+describe('chainer', () => {
+  it('should be a function', () => {
+    expect(chainer).toBeInstanceOf(Function);
+  });
+
+  it('should return second arg if arr of func is empty', () => {
+    expect(chainer([])(2)).toBe(2);
+  });
+
+  it('should return correct result', () => {
+    const a = jest.fn((x) => x);
+    const b = jest.fn((x) => x + 2);
+    const c = jest.fn((x) => x - 1);
+
+    expect(chainer([a, b, c])(2)).toBe(3);
+  });
+
+  it('should call every function in function array', () => {
+    const a = jest.fn((x) => x);
+    const b = jest.fn((x) => x + 2);
+    const c = jest.fn((x) => x - 1);
+
+    chainer([c, b, a])(10);
+
+    expect(a).toHaveBeenCalled();
+    expect(b).toHaveBeenCalled();
+    expect(b).toHaveBeenCalled();
```

Comment:
There is a duplicate expectation for `expect(b).toHaveBeenCalled();`. This line should be removed or replaced with the correct expectation for function `c` to ensure all functions are tested for being called.